src/utils/zipcodeConversion.js

```diff
@@ -0,0 +1,15 @@
+import zipcodes from 'zipcodes';
+
+const zipcodeConversion = zipcode => {
+  try {
+    let convertedZipcode = zipcodes.lookup(zipcode);
+    let error =
+      'Sorry, zipcode is invalid. Please check your input and try again.';
+    if (!convertedZipcode) throw error;
```

Comment:
Should this not throw an actual error that also contains the zipcode that caused the error.
```js
if (!convertedZipcode) throw new Error(`Zipcode "${zipcode}" is invalid.`)
```